package.json

```diff
@@ -9,11 +9,11 @@
     "clean": "rimraf packages/*/dist/ packages/*/node_modules/ packages/*/coverage/ packages/*/umd/",
     "build-api-docs": "lerna run build-api-docs",
     "format-all": "prettier --write \"packages/*/src/**\" \"packages/*/__tests__/**\"",
-    "lerna-audit": "lerna-audit",
+    "lerna-audit-production": "lerna exec \"npm audit --production\"",
```

Comment:
Why production and not also devdeps?

Reply:
I'm not sure if that's a good idea - devDependencies can still impact downstreams. For example, if one of our build tools is compromised, it can inject code into the production bundle. Or if it's used to compromise a developer machine or our CI environment, it might get hold of credentials that can be used to push code to downstreams.

Reply:
After discussing with Nic elsewhere, the real issue here is that `npm audit` fails on an issue in a dependency for which no fix is available yet, and it's not possible to set it up such that it only fails when there's actually something we can do to resolve it.

Thus, I think that if we mark this explicitly as a temporary workaround to deal with that specific issue, and when it can be reverted, then applying this is fine.